packages/web-app/lib/constants/sepolia.tsx

```diff
@@ -0,0 +1,10 @@
+import { BigNumber } from "ethers";
+
+export const permit2Address = "0x000000000022D473030F116dDEE9F6B43aC78BA3";
+export const NATION = "0x23Ca3002706b71a440860E3cf8ff64679A00C9d7";
+export const frameworkAddress = "0xD96aA6e2568f4e9632D2A5234Bb8410ca7609a27";
+export const arbitratorAddress = "0xBe67cEdCD1FE38aac8a5781A51250FDeFB344E6C";
+export const cohortAddress = "0xfbb66bc799308435ed2a0e0c0ac3ad1d46749b7b";
+export const safeTxServiceUrl = "https://safe-transaction-sepolia.safe.global";
+export const subgraphURI = process.env.NEXT_PUBLIC_GRAPH_API_URL_SEPOLIA;
```

Comment:
```suggestion
export const subgraphURI = process.env.NEXT_PUBLIC_GRAPH_API_URL;
```

Reply:
Does this mean that https://agreements.nation3.org/ supports only the Ethereum mainnet, and https://sepolia.agreements.nation3.org/ supports only the Sepolia testnet, correct?

Reply:
we don't support network switching?

Reply:
> we don't support network switching?

We can still support it, by redirecting to the other URLs.  If a user opens https://agreements.nation3.org and selects Sepolia in the chain selection drop-down, we can redirect to https://agreements-sepolia.nation3.org